sano2019/rent-a-pet
# Reshape pet model

I did:
1. pet model wit the new columns, associations and validation(we can add more validations later on)  
2. seeds
3. in user controller I added dependent: : destroy for pets
how to test:
"1" you can pull the code, run the seed and try email: "woland@sisi.com", password: "123456"
look at the code and at schema
"2" run the seeds
"3" run the seeds several times, should have no errors
